osu.Game/Overlays/Login/LoginPanel.cs

```diff
@@ -139,6 +139,8 @@ private void load()
                         },
                     };
 
+                    api.LocalUser.Value.Status.BindValueChanged(e => updateDropdownCurrent(e.NewValue), true);
```

Comment:
i'll fix this, but note that as you didn't create a local bindable this could cause a bindable leak. only saved by the fact there's only ever one `LoginPanel`.

Reply:
gotcha, will keep in mind